api/app/utils/time.py

```diff
@@ -16,6 +16,15 @@ def get_utc_now():
     return datetime.now(tz=timezone.utc)
 
 
+def get_utc_today_start_and_end():
```

Comment:
Why are we concerned with start and end in UTC? Shouldn't we be interested in start and end of the day in PST?

Reply:
Wasn't sure so I went for the most agnostic option.

Reply:
Yeah - you're going to need to do something like
`
today = get_pst_now()
# where PST_TO_UTC is some magic sauce that I don't know....
start = PST_TO_UTC(today.replace(hour=0, minute=0, second=0, microsecond=0))
end = start + timedelta(days=1)


`

Reply:
It's easy with `pytz` but I think we removed that dependency

Reply:
Oh yeah - never mind that last turning it back into utc thing - I think this is good.

Reply:
```suggestion
def get_pst_today_start_and_end():
```
Pls. rename function, and maybe add some comments around why it has to be in PST.